core/services/workflows/engine.go

```diff
@@ -1300,6 +1301,10 @@ func NewEngine(ctx context.Context, cfg Config) (engine *Engine, err error) {
 	workflow.hexName = hex.EncodeToString([]byte(cfg.WorkflowName))
```

Comment:
is it simpler to not add the `WorkflowNameTransform` field (it's public and could get messed with in theory) and instead just do:

```
workflow.hexName = pkgworkflows.HashTruncateName(cfg.WorkflowName)
```

Reply:
That's what I originally tried to do, but as @cedric-cordenier caught [here](https://github.com/smartcontractkit/chainlink/pull/15896#pullrequestreview-2546244614) that would cause a breaking change for the existing Keystone deployments